src/SingleCellAnnotator.imjoy.html

```diff
@@ -473,9 +479,18 @@
       });
       api.alert(JSON.stringify(blob))
     },
+    // go to nextImage after select yes/no.
+    async selectLocation(judgement){
+      if (judgement===true){
+      this.current_location_selected = true;}
+      if (judgement===false){
```

Comment:
`judgement` ==> `selected`
and you should use `else` for the other case.

Reply:
done